README.md

```diff
@@ -1,10 +1,30 @@
-# free5GC Compose
+# Free5GC Compose
 
 This repository is a docker compose version of [free5GC](https://github.com/free5gc/free5gc) for stage 3. It's inspire by [free5gc-docker-compose](https://github.com/calee0219/free5gc-docker-compose) and also reference to [docker-free5GC](https://github.com/abousselmi/docker-free5gc).
 
 You can change your own config in [config](./config) folder and [docker-compose.yaml](docker-compose.yaml)
 
-## Pre-request
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
Do we need to install any tool to use TOC auto update?

Reply:
The TOC is generated using the doctoc (https://github.com/thlorenz/doctoc) tool. To use it without intalling anything (except docker) I use this docker image: https://hub.docker.com/r/jorgeandrada/doctoc/

I use this command at the root of the project:

```console
docker run --rm -it -v $(pwd):/usr/src jorgeandrada/doctoc
```
It will generate TOC for any markdown file that it finds in the PWD and its subdirs.